src/py42/_internal/client_factories.py

```diff
@@ -94,6 +98,18 @@ def _get_stored_value(self, key):
             self._key_value_store_client = KeyValueStoreClient(session)
         return self._key_value_store_client.get_stored_value(key).text
 
+    def get_high_risk_employee_client(self):
+        # TODO figure out the correct session instance to be created.
+        session = ""
+        if not self._high_risk_employee_client:
+            self._high_risk_employee_client = HighRiskEmployeeClient(session, self._user_context)
+
+        # TODO similarly detection_list too but since there is no property to instantiate it, need
```

Comment:
this client would also use `session = self._get_jwt_session(u"employeecasemanagement-API_URL")`

Reply:
Here I meant to ask, When will `DetectionListUserClient be` instantiated? 
In this method only `HighRiskEmployeeClient` should be instantiated.

since we directly call `sdk.detectionlist.update_notes` when will `self._detection_list_user_client` be defined?

Whereas for `self.detectionlist.high_risk_employee` will define `self._high_risk_employee_client`.

Reply:
`get_detection_list_user_client` should be its own function though

Reply:
Done